### PR TITLE
Work on translating PrivacyManager templates

### PR DESCRIPTION
**Pull request by maxpereira**
Work to fix issue #14954

### CODE REVIEW

**Comment by Findus23** [comment]
Hi,

Thanks for the work and just a quick note:
I know little about angular, but I think `html_attr` should only be used when the escaped text is put in an attribute (so like in `content-title`).

File: plugins/PrivacyManager/templates/askingForConsent.twig [hunk 1]
```diff
@@ -4,48 +4,49 @@
...
-            Note that when you require to have your users' consents and if you cannot prove that you got the "Consent" of your users in the past, then we recommend to <a href="{{ linkTo({module: 'PrivacyManager', action: 'privacySettings'}) }}">anonymise your previously tracked users' raw data</a>.
+            {{ 'PrivacyManager_WhenINeedConsentNote2'|translate|e('html_attr') }} <a href="{{ linkTo({module: 'PrivacyManager', action: 'privacySettings'}) }}">anonymise your previously tracked users' raw data</a>.
         </p>
     </div>
 
-    <div piwik-content-block content-title="How do I ask users for consent?" class="privacyAskingForConsent">
+    <div piwik-content-block content-title="{{ 'PrivacyManager_HowToAskConsent'|translate|e('html_attr') }}" class="privacyAskingForConsent">
         <p>
-            Follow the steps below to ask your user for their consents before their data is processed within Matomo.
+            {{ 'PrivacyManager_HowToAskConsentNote1'|translate|e('html_attr') }}
```
Comment (tsteur) [inline plugins/PrivacyManager/templates/askingForConsent.twig]: @maxpereira FYI you only need `|e('html_attr')` when it is part of an HTML attribute like `content-title="..."` above. In here for example you should remove `|e('html_attr')`

**Comment by tsteur** [comment]
Awesome @maxpereira very much appreciated. Be great to ping me once you're ready for a review

**Comment by maxpereira** [comment]
Thanks for the tip! I’ll continue working on this and make the necessary changes.

**Comment by sgiehl** [comment]
ping @maxpereira  Any update on this one? Are you still working on it?

**Comment by sgiehl** [comment]
The source repository has been deleted meanwhile, so guess there won't be any updates to this PR.

**Pull request closed without merging**